examples/text-generation/README.md

```diff
@@ -474,17 +474,19 @@ Below example uses `flash_attention_recompute` mode in order to reduce memory co
 python ../gaudi_spawn.py --use_deepspeed --world_size 8 run_generation.py \
 --model_name_or_path meta-llama/Llama-2-70b-hf \
 --use_hpu_graphs \
+--limit_hpu_graphs \
 --use_kv_cache \
---reuse_cache \
+--bf16 \
 --trim_logits \
 --attn_softmax_bf16 \
---max_input_tokens 31744 \
---max_new_tokens 1024 \
---batch_size=12 \
+--bucket_size=128 \
+--bucket_internal \
+--batch_size 10 \
+--max_input_tokens 40960 \
+--max_new_tokens 5120 \
 --use_flash_attention \
 --flash_attention_recompute \
---flash_attention_causal_mask \
---book_source
+--flash_attention_causal_mask
```

Comment:
The outputs I get with this command are weird:
```
Input/outputs:                                                                                                                               
input 1: ('DeepSpeed is a machine learning framework',)                                                                                      
output 1: ('DeepSpeed is a machine learning framework for\n\n\n\n\n\n\n\n\n                                                                  
                                                                                                                                             
                                                                                                                                             
                                           a           a                    a                        a                 a       a             
 a               a                a    a        a       a        a        a        a             a      a          a        a        a       
 a  a       a       a      a       a      a      a       a   a    a     a    a     a    a  a      a      a     a       a   a    a       a    
a      a      a            a        a   a      a    a     a         a   a   a       a     a        a    a  a   a  a     a   a   a   a  a  a  
 a     a    a   a        a       a      a    a    a   a   a   a    a  a  a   a   a  a  a     a  a  a       a   a     a    a  a     a   a   a 
  a        a   a   a    a   a      a    a   a   a       a  a    a   a      a      a    a    a   a   a    a   a     a   a    a   a         a  
  a   a   a  a  a  a          a  a   a  a  a  a    a    a  a   a   a    a   a   a       a  a        a     a   a   a    a   a    a   a  a   a 
   a   a    a       a  a   a    a    a   a  a   a  a      a     a    a       a   a       a    a  a       a  a    a  a   a       a    a  a  a 
 a         a       a    a   a   a  a  a    a  a     a       a       a   a  a  a  a    a   a          a   a   a     a   a        a   a  a    a
     a   a  a   a    a  a  a   a    a    a           a    a  a     a   a        a   a    a   a     a   a    a     a    a   a    a   a   a    
     a   a  a    a       a     a    a   a    a   a    a     a   a    a   a     a  a   a    a   a    a   a    a         a  a  a   a      a   a
   a    a         a  a    a   a   a   a    a     a      a    a    a   a  a  a  a    a    a  a  a    a  a  a   a   a  a   a  a  a  a    a   a 
    a   a    a  a  a    a    a    a  a  a    a     a     a  a    a   a   a   a        a   a    a  a  a     a   a    a  a   a  a      a  0    
a    a  a   a  a 0        a  a    a    a    a  a    a  a  0  a  a    a   a   a       a  a  a  a  a   a  a  a   a  a  a     a  a  a  a  a  a  
  a  a  a  a   a  a    a   a   a     a   a    a   a   a  a  a   a    a  a  a    a    a  -  -     a  a  a 0  0     a  a   a  - 0      a  a 0  
0   0        a    a   and -      a   a   a - 0       0   0    0      0  0     0      a   a 0  0      0  0   0         0  0   0      0 0   0  
 0    0 0  0   0   0     0    0   0       a 0  0  0    0  0   0  0        a 0  0  0       0  0    0     0  0  0  0         0   0   0     0  0
   0  0       0   0     0  0 0   0  0         0   0      0   0  0   0      0   0    0    0   0  0    0       0   0    0      0   0    0    0 
 0   0    0     0  0   0      0 0  0    0     0  0  0      0      0    0       0 0  0   \\\\       0   0   0      0  0     0      0 0     0  
  0    0      0     0  0   0     0   0    0    0   0  0   0     0    0  0   \\\\  \\\\          0      0    0 0    0        0 0  0        0 0
   0     0   0  0   0      0    0  0      0   0   0     0     0  0    0     0  0  0      0     0 0     0      0  0    0     0  0 0   0       
0   0      0   0 0   0  0    \\\\  0    0  \\\\   \\\\ \\\\          0    0 0   0 \\\\  \\\\ \\\\ \\\\      \\\\   \\\\ \\\\ \\\\   \\\\ \\\\
 \\\\  \\\\  \\\\ \\\\     \\\\ \\\\ \\\\ \\\\ \\\\ \\\\      \\\\ \\\\ \\\\   \\\\ \\\\   \\\\ \\\\  \\\\ \\\\ \\\\ \\\\     \\\\  \\\\  \\\
\ \\\\ \\\\ \\\\    \\\\   \\\\ \\\\ \\\\ \\\\  \\\\ \\\\ \\\\ \\\\ \\\\ \\\\ \\\\ \\\\ \\\\ \\\\\\\\\\\\\\\\\\\\ \\\\\\\\\\\\\\\\\\\\\\\\ \\
\\ \\\\ \\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\ \\\\ \\\\ \\\\ \\\\ \\\\ \\\\\\\\ \\\\ \\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\
\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\
\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\ \\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\    \\\\ \
\\\ \\\\ \\\\ \\\\\\\\\\\\ \\\\ \\\\ \\\\ \\\\ \\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\ \\\\ \\\\ \\\\ \\\\ \\\\ \\\\ \\\\   \\\\ \\\\ \\\\ \\\\ \\\\
\\\\\\\\\\\\\\\\\\\\\\\\ \\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\
```

Reply:
Thank you for reviewing. I will check this out.